providers/common/include/internal/providercommonerr.h

```diff
@@ -95,6 +95,7 @@ int ERR_load_PROV_strings(void);
 # define PROV_R_UNABLE_TO_LOAD_SHA1                       143
 # define PROV_R_UNABLE_TO_LOAD_SHA256                     147
 # define PROV_R_UNSUPPORTED_CEK_ALG                       145
+# define PROV_R_UNSUPPORTED_KEY_SIZE                      150
```

Comment:
Remove this as well before `make update`

Reply:
Will fix after dependent PR's are in.